### ISSUE TITLE
Issue: installing vampnet

### ISSUE THREAD

**Issue by dlbeveridge** [markovmodel/deeptime#12]
HAve been proceeding as instructed with the installation of tensorflow and vampnet.

I followed the install tensorflow  on Windows 7 using anaconda. This had me create a tensorflow environment, and install it there. Followed the instructions on the tensorflow website and this worked fine.  The tensorflow program Hello ... worked fine.

Now I am trying to install vampnet. Did a github clone of deeptime-master onto my desktop. So, still with the tensorflow environment  activated I cloned the git hub deeptime-master, which looked like a complete github respositoty (I think - 3 git.. files are included, plus a vampnet directory.) So went into vampnet directory and ran  python setup .py  install.

This failed on  line 339 in the setup.py and also gave me: LookupError: setuptools-scm was unable  to detect versions for 'C:  .....  \\deeptime-master'

Same exaact thing happened when I tried to install tensorflow and vampnet on my macbook pro.

Can you give me any help with this?

Thanks

DLB

**Comment by Obudo** [comment]
I couldn't replicate the issue, will do more tests. In the meantime, could you try running

python setup.py develop

Instead?

**Comment by amardt** [comment]
Hey,

I think this is an issue caused by the fact that this repository handles two packages (on my linux machine everything works fine). Anyway, I am not confident that I understood your Error exactly. 
Can you give us more inside? (the result of a conda list command. ) 
Also can you try out-comment by # the lines in the setup.py file which evoke setuptools-scm:
Should be the following lines
use_scm_version=dict(root='..', relative_to=__file__)
setup_requires=['setuptools_scm', 'setuptools_scm_git_archive'].

Additionally, it would be nice to have your whole error message. I don't know in which file the line 339 should be. 

Best,
Andreas

**Comment by dlbeveridge** [comment]
Luca, Andreas,

Thanks for the suggestions. Meanwhile here is what I got on the terminal (MAC OS):


$source activate tensorflow
(tensorflow) $pwd
/Users/dbeveridge/Desktop/deeptime-master/vampnet
(tensorflow) $ls
README.md	examples	setup.py	vampnet
(tensorflow) $ python setup.py install
Traceback (most recent call last):
  File "setup.py", line 39, in <module>
    zip_safe=False)
  File "//anaconda/envs/tensorflow/lib/python3.5/site-packages/setuptools/__init__.py", line 129, in setup
    return distutils.core.setup(**attrs)
  File "//anaconda/envs/tensorflow/lib/python3.5/distutils/core.py", line 108, in setup
    _setup_distribution = dist = klass(attrs)
  File "//anaconda/envs/tensorflow/lib/python3.5/site-packages/setuptools/dist.py", line 372, in __init__
    _Distribution.__init__(self, attrs)
  File "//anaconda/envs/tensorflow/lib/python3.5/distutils/dist.py", line 281, in __init__
    self.finalize_options()
  File "//anaconda/envs/tensorflow/lib/python3.5/site-packages/setuptools/dist.py", line 528, in finalize_options
    ep.load()(self, ep.name, value)
  File "/Users/dbeveridge/Desktop/deeptime-master/vampnet/.eggs/setuptools_scm-2.0.0-py3.5.egg/setuptools_scm/integration.py", line 22, in version_keyword
    dist.metadata.version = get_version(**value)
  File "/Users/dbeveridge/Desktop/deeptime-master/vampnet/.eggs/setuptools_scm-2.0.0-py3.5.egg/setuptools_scm/__init__.py", line 119, in get_version
    parsed_version = _do_parse(root, parse)
  File "/Users/dbeveridge/Desktop/deeptime-master/vampnet/.eggs/setuptools_scm-2.0.0-py3.5.egg/setuptools_scm/__init__.py", line 97, in _do_parse
    "use git+https://github.com/user/proj.git#egg=proj" % root)
LookupError: setuptools-scm was unable to detect version for '/Users/dbeveridge/Desktop/deeptime-master'.

Make sure you're either building from a fully intact git repository or PyPI tarballs. Most other sources (such as GitHub's tarballs, a git checkout without the .git folder) don't contain the necessary metadata and will not work.

For example, if you're using pip, instead of https://github.com/user/proj/archive/master.zip use git+https://github.com/user/proj.git#egg=proj
(tensorflow) $

**Comment by dlbeveridge** [comment]
Andreas and Luca,

Finally succeeded in getting a vampnet installation to install and perform the 1D_DBLWELL calculation.

Andreas, your suggestion about commenting out the two lines in the vampnet setup.py and running resulting in a successful installation. Note I am running everything in an anaconda environment "tensorflow".  

Trying to run from the jupyter notebook has not been successful yet.  My problem has to do with the proper installation of python backend and the web-based nature of notebooks. Whatever, I put that aside for the moment and moved the commands in the 1D_DBLWELL notebook over into a single .py which I could run from my terminal (Still tensorflow env).  

I had to fuss with the install step but with some advice from Google I got this running. The rest of the commands executed in my independent .py. This (eventually) worked and produced all the plots successfully, so I am pretty happy about that. 

Now need to figure out my problem with the notebook version - haven't got back to that, but would appreciate any suggestions. It is like the python the notebook is looking at does not have the right backend capability, but need to figure out how to fix this.

Whatever, thanks very much for your help.  Congratulations to you all and Frank on the Vampnet project and Nature paper. This is an historic accomplishment, with parallels to the Hartree-Fock variational optimization of one-electron wavefumctions in QM molecular orbital theory. 

DLB
